src/test/java/nextstep/step3/domain/RacingCarsTest.java

```diff
@@ -3,15 +3,41 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import java.util.Arrays;
+import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class RacingCarsTest {
 	@Test
 	@DisplayName("자동차수만큼 레이싱카를 만든다.")
 	public void racingCarsTest() {
-		RacingCars racingCars = new RacingCars(Arrays.asList("adel","joy","ruby"));
+		RacingCars racingCars = new RacingCars("adel,joy,ruby");
 		assertThat(racingCars.getRacingCars().size()).isEqualTo(3);
 	}
+
+	@Test
+	@DisplayName("레이싱에 우승한 자동차명을 얻을수 있다.")
+	public void racingCarWinnersTest() {
+		RacingCars racingCars = new RacingCars("adel,joy,ruby");
+		int point = 1;
+		for (RacingCar car : racingCars.getRacingCars()) {
+			car.move(++point);
+		}
+		List<String> winners = racingCars.getRacingWinnerNames();
+		assertThat(String.join(",", winners)).isEqualTo("ruby");
+	}
```

Comment:
마지막 미션도 리팩토링을 잘해주셔서 추가적으로 리팩토링 거리를 하나 더 드립니다 ㅎㅎ
다음과 같은 테스트가 통과할 수 있도록 리팩토링 해보시면 좋을 것 같아요!
```suggestion
	public void racingCarWinnersTest() {
		RacingCars racingCars = new RacingCars("adel,joy,ruby");
		RacingCars expect = new RacingCars("ruby");
                 int point = 1;
                 
		for (RacingCar car : racingCars.getRacingCars()) {
			car.move(++point);
		}
		
		RacingCars winners = racingCars.getRacingWinnerNames();
		assertThat(winners).isEqualTo(expect);
	}
```

Reply:
요건 수정하지 않았습니다. 
제가 의도했던 `getRacingWinnerNames() ` 결과는 최종 출력하기 위한 우승자 이름 목록만 뽑는거예요.
위처럼 RacingCars를 리턴받았을때 다시 자동차 이름목록을 만들어야 하므로 현재 로직을 유지할께요. 🙏

Reply:
그런 의도셨군요!
저는 데이터를 비교하는 테스트도 좋지만 객체 지향인 만큼 객체간 논리적인 동치성을 비교해도록 테스트하는게 의도였습니다 ㅎㅎ
말씀해주신대로 현재 로직을 유지하시면 좋을 것 같아요 👍